src/mmw/js/src/core/models.js

```diff
@@ -239,14 +239,10 @@ var AnimalCensusCollection = Backbone.Collection.extend({
 
 var PointSourceCensusCollection = Backbone.PageableCollection.extend({
```

Comment:
These `Backbone.PageableCollection` calls are throwing errors now since the library has been removed.